driver-core/src/test/java/com/datastax/driver/core/OPPTokenFactoryTest.java

```diff
@@ -45,20 +45,6 @@ public void should_split_range_producing_empty_splits_near_ring_end() {
     assertThat(splits).containsExactly(zero, zero);
   }
 
-  @Test(groups = "unit")
-  public void should_strip_trailing_0_bytes() {
-    Token with0Bytes = token(ByteBuffer.wrap(new byte[] {4, 0, 0, 0}));
-    Token without0Bytes = token(ByteBuffer.wrap(new byte[] {4}));
-    Token fromStringWith0Bytes = factory.fromString("040000");
-
-    assertThat(with0Bytes).isEqualTo(without0Bytes).isEqualTo(fromStringWith0Bytes);
-
-    Token withMixed0Bytes = factory.fromString("0004000400");
-    Token withoutMixed0Bytes = factory.fromString("00040004");
-
-    assertThat(withMixed0Bytes).isEqualTo(withoutMixed0Bytes);
-  }
-
```

Comment:
Can we add a new test method to cover the additional `start.equals(end)` check?
For example:
```java
factory.split(
    token(Bytes.fromHexString("0x11")),
    token(Bytes.fromHexString("0x1100")),
    whatever)
```
should yield `0x1100` as the split boundaries.

Reply:
done 👍